config/optional/honeypot.settings.yml

```diff
@@ -0,0 +1,16 @@
+unprotected_forms:
```

Comment:
Should this be in `config/optional`? I'm not totally over this but isn't stuff in this directory only if it is _optionally_ installed based on if dependencies are met? What is the optional dependency here?

Reply:
Putting it in config/install gives error during build because after honeypot gets installed it couldn't replace this settings file. So I put it in the optional folder and when the dependency gets installed in content vic or any other site we can customise it accordingly. I might be wrong in this. Should I just override this settings in the update_hook rather than put it in the optional folder!!